### DISCUSSION TITLE
Regarding the issue with the "se_e2_a" and "se_e3" models, seeking advice.

### DISCUSSION THREAD

**Discussion by composites123** [deepmodeling/deepmd-kit#3044] [Q&A] [answered]
Hello everyone,
I'm currently using the **"se_e2_a"** model to train a force field that describes phase transitions in **carbon materials**. However, I've learned from the documentation that the **"se_e3"** model is better suited for describing the three-body interactions in carbon. Unfortunately, after trying it out, I've found that our current computational resources within the group are insufficient to support the use of the "se_e3" model. **My question is, can I still obtain a relatively good force field using the "se_e2_a" model?**

**Comment by wanghan-iapcm** [comment, marked as answer]
you may try to use se_e3 with a smaller cutoff radius and hybrid it with a se_e2_a with a normal cut-off radius .

**Reply by composites123** [reply]
Thank you very much for your guidance. The method you suggested was highly effective; I reduced the rcut of se_e3 to 5.0, significantly reducing the GPU memory required for training.

Additionally, I have a question I'd like to ask you. For carbon materials, such as graphene, the generation of interlayer covalent bonds usually occurs at an interlayer distance of 2.0. The advantage of the se_e3 model lies in its ability to describe more complex bond angle changes. **Would it be appropriate for me to directly set the rcut of se_e3 to 3.0? This way, it would focus on describing their more complex interactions only when the carbon-carbon distance reduces to within 3.0.**

Thank you for your assistance.

**Reply by wanghan-iapcm** [reply]
It is fully valid. Usually the se_e3 is only needed for nearest neighbors.

**Reply by composites123** [reply]
I understand now. Thank you very much for your guidance.
